css/excercise_one/part-c/scss/excercise_one.scss

```diff
@@ -0,0 +1,184 @@
+.px-xl { @include padding(null, 120px, null, 120px); }
+.py-sm { @include padding(8px, null, 8px); }
+
+.m-0 { @include margin(0px, 0px, 0px, 0px); }
+.mt-0 { @include margin(0px); }
+.mt-sm { @include margin(8px); }
+.mt-md { @include margin(20px); }
+.mt-lg { @include margin(70px); }
+.mt-xl { @include margin(110px); }
+.mt-xxl { @include margin(300px); }
+.mt-ng { @include margin(-20px, 0px, 0px, 0px); }
+.mr-sm { @include margin(null, 8px); }
+.mr-lg { @include margin(null, 70px); }
+.mb-0 { @include margin(null, null, 0px); }
+.mb-xs { @include margin(null, null, 8px); }
+.mb-sm { @include margin(null, null, 20px); }
+.mb-md { @include margin(null, null, 50px); }
+.mb-lg { @include margin(null, null, 70px); }
+.mb-xl { @include margin(null, null, 80px); }
+.mb-ng { @include margin(0px, 0px, -65px, 0px); }
+.ml-sm { @include margin(null, null, null, 8px); }
+.ml-md { @include margin(null, null, null, 20px); }
+.ml-lg { @include margin(null, null, null, 70px); }
+.my-lg { @include margin(200px, null, 200px); }
+
+.text-right { text-align: right; }
+.text-center { text-align: center; }
+.text-left { text-align: left; }
+
+.text-uppercase { text-transform: uppercase; }
+
+.font-weight-light { font-weight: map-get($map: $font-w, $key: 'light') }
+.font-weight-medium { font-weight: map-get($map: $font-w, $key: 'medium') }
+.font-weight-bold { font-weight: map-get($map: $font-w, $key: 'bold') }
+.font-weight-extra-bold { font-weight: map-get($map: $font-w, $key: 'xbold') }
+
+.text-xs { font-size: map-get($map: $font-s, $key: 'xsmall') }
+.text-sm { font-size: map-get($map: $font-s, $key: 'small') }
+.text-md { font-size: map-get($map: $font-s, $key: 'medium') }
+.text-lg { font-size: map-get($map: $font-s, $key: 'large') }
+.text-xl { font-size: map-get($map: $font-s, $key: 'xlarge') }
+.text-xxl { font-size: map-get($map: $font-s, $key: 'xxlarge') }
+.text-xxxl { font-size: map-get($map: $font-s, $key: 'xxxlarge') }
+
+.line-height-sm { line-height: 19.5px; }
+.line-height-md { line-height: 37.8px; }
+.line-height-lg { line-height: 52px; }
+.line-height-xlg { line-height: 65px; }
+
+.d-inline-block { display: inline-block; }
+.d-flex { display: flex; }
+.flex-column { flex-direction: column; }
+.justify-content-start { justify-content: flex-start; }
+.justify-content-center { justify-content: center; }
+.justify-content-end { justify-content: flex-end; }
+.justify-content-between { justify-content: space-between; }
+.justify-content-around { justify-content: space-around; }
+.align-items-center { align-items: center; }
+
+.position-relative { position: relative; }
+.position-absolute { position: absolute; }
+
+.top-0 { top: 0; }
+.right-0 { right: 0; }
+.bottom-0 { bottom: 0; }
+.bottom-20px { bottom: 20px; }
+.left-0 { left: 0; }
+.left-365px { left: 365px; }
+
+.rounded { border-radius: 5px; }
+.top-rounded { border-top-left-radius: 20px; border-top-right-radius: 20px;}
+.ls-negative { letter-spacing: -1px; }
```

Comment:
I would not abbreviate here since letter spacing is not such a straight forward property name